src/components/Connection.tsx

```diff
@@ -519,9 +523,9 @@ const Connection: React.FC<ConnectionProps> = ({
           <div className="flex items-center space-x-2">
             {detectedBits ? (
               <Button
-                className={`w-36 flex justify-center items-center overflow-hidden ${
+              className={`w-36 flex justify-center items-center overflow-hidden ${
                   selectedBits === "auto"
-                    ? "bg-dark text-light"
+                    ? "bg-primary text-light"
                     : "bg-white text-black"
```

Comment:
bg-primary is a default property should not add that in className, also make sure to not use bg-white or bg-black, we have defined colors in global css, for white bg-foreground is there and for black bg-background. Defined colors are not exactly black and white but a shade of it and we are using them in whole app.

Reply:
For autoscale button shift, just use ternary operator and toggle between default and outline button variant. we don't need to define color of anything in this

Reply:
<Button
                variant= {selectedBits === "auto"? "outline" : "default"}
                className={`w-36 flex justify-center items-center overflow-hidden`}
                onClick={() =>
                  setSelectedBits(
                    selectedBits === "auto" ? detectedBits : "auto"
                  )
                }
              >
                Autoscale
              </Button>
              
              
              This will be the button